presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -160,6 +163,32 @@ public void testArrayElements()
         assertInvalidFunction("CAST(ARRAY ['puppies', 'kittens'] AS ARRAY<BIGINT>)", INVALID_CAST_ARGUMENT);
     }
 
+    private static String generateLongSring(int size, char c)
+    {
+        char[] cs = new char[size];
+        Arrays.fill(cs, c);
+
+        return new String(cs);
+    }
+
+    private static String generateThreeElementArrayString(int size)
```

Comment:
Inline this "generateThreeElementArrayString" function. 

`generateThreeElementArrayString` is not an accurate depiction of what this function does. As a result, I don't think this provides "comment" value anyways.